docs/mkdocs.yml

```diff
@@ -1,10 +1,120 @@
-site_name: Catalyst CI Documentation
+# cspell: words Diagramsnet, pymdownx, linenums, inlinehilite, superfences,
+# cspell: words tasklist, betterem, materialx, twemoji, smartsymbols,
+# cspell: words kroki, excalidraw, glightbox, cips, pygments, arithmatex
+
+# Project Information
+site_name: Project Catalyst - Catalyst CI Documentation
```

Comment:
On my end, this looks pretty ugly. The name is too long and wraps in the navigation bar. Maybe something in the middle, like: "Project Catalyst CI Docs" ?

Reply:
No problem.

Reply:
Done